File: Java/escaneo/src/main/java/com/sistema/dto/UsarEntradaRequest.java
package com.sistema.dto;

/**
 * Clase DTO (Data Transfer Object) para manejar las solicitudes de usar una entrada.
 */
public class UsarEntradaRequest {

    // Identificador de la entrada a ser usada
    private int entradaId;

    // Getters y setters

    /**
     * Obtiene el identificador de la entrada.
     *
     * @return el identificador de la entrada
     */
    public int getEntradaId() {
        return entradaId;
    }

    /**
     * Establece el identificador de la entrada.
     *
     * @param entradaId el nuevo identificador de la entrada
     */
    public void setEntradaId(int entradaId) {
        this.entradaId = entradaId;
    }
}

File: Android/app/src/main/java/com/example/appeventos/model/UsarEntradaRequest.kt
package com.example.appeventos.model

// Definición de la clase de datos UsarEntradaRequest
data class UsarEntradaRequest(
    val entradaId: Int, // Identificador de la entrada que se desea usar
    val usuarioId: Int // Identificador del usuario que desea usar la entrada
)
